### DISCUSSION TITLE
how to use oscp for validating certificate file in the docker install environment

### DISCUSSION THREAD

**Discussion by lu-guangrui** [Keyfactor/ejbca-ce#132] [Q&A]
Hi,
I'm new to EJBCA, openSSL, ca and others in the security system . so I installed EJBCA  following this [page](https://doc.primekey.com/ejbca/tutorials-and-guides/quick-install-guide) for testing and evaluation purposes，and now it is working in my local.I can access the administrator web, ra web , 
![image](https://user-images.githubusercontent.com/17984591/196425804-900916be-858f-48be-b64d-12bc2f8d26a1.png)
![image](https://user-images.githubusercontent.com/17984591/196425870-1033cd14-4f18-4708-ac0a-43ab29ea7414.png)

then I added one end entity and generated it's ca file, the next  step what I want to do is : revoke the ca and ues the ocsp to validate the ca file ,but I don't konw how , I tried following the [page](https://doc.primekey.com/ejbca/ejbca-operations/ejbca-ca-concept-guide/protocols/ocsp),but failed...
when I access https://myhost/ejbca//ejbca/publicweb/status/ocsp/MEkwRzBFMEMwQTAJBgUrDgMCGgUABBT1x9iHOmegR7sYp5Fzdo/u3FMBRgQUH1DTnzl9lscAe3WBTUPCWR+xwpQCCDlpiyU2q5Rb，the browser return nothing but a file，which named u3FMBRgQUH1DTnzl9lscAe3WBTUPCWR+xwpQCCDlpiyU2q5Rb ,had no type suffix and its content is confusing , like this 
![image](https://user-images.githubusercontent.com/17984591/196425101-4c401e31-9146-4154-a5c3-a5075e205a89.png)

so I need a more detailed instruction manual, please give me some help .
Thanks.

**Comment by primetomas** [comment]
OCSP is specified in the RFC6960 standard. https://www.rfc-editor.org/rfc/rfc6960
That is nothing you can use as a URL in the browser. You need an OCSP client to validate with OCSP.

See the documentation for example commands using OpenSSL. https://doc.primekey.com/ejbca/ejbca-operations/ejbca-ca-concept-guide/protocols/ocsp
